internal/utils/utils.go

```diff
@@ -22,16 +22,8 @@ import (
 	"github.com/cilium/cilium-cli/defaults"
 )
 
-func BuildImagePath(userImage, defaultImage, userVersion, defaultVersion string) string {
-	if userImage == "" {
-		userImage = defaultImage
-	}
-
-	if userVersion == "" {
-		userVersion = defaultVersion
-	}
-
-	return userImage + ":" + userVersion
+func BuildImagePath(image, version string) string {
+	return image + ":" + version
```

Comment:
Nyc, but since you are making changes anyway, I'll post my comment here. The version string shouldn't be appended to a custom image.

Reply:
Looks like the correct format to pass a custom image is - `--agent-image quay.io/foo/ --version bar`. This will then be computed as `quay.io/foo:bar`. This isn't intuitive! @kaworu Could you add an example to the help dump?

Reply:
Sounds good to me! Thanks!